### PR TITLE
Add COMIT as an atomic swap protocol

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 1]
```diff
@@ -82,6 +83,13 @@ as a third party. Although swaps have been done, it is stated as not yet being p
 (July&nbsp;2018), it is only possible to use Barterdex out of the Command Line Interface (CLI) [[9]]. Barterdex charges 
 a 0.1287% fee for a swap [[10]]. 
 
+### #4 COMIT
+COMIT is an open protocol for atomic swaps [[13]]. Compared to other solutions, COMIT does not feature another _blockchain_ nor token. Powered by simple cryptographic principles such as HTLCs [[12]] it enables users to trustlessly exchange one digital asset to another.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
Cryptographically-secure Off-chain Multi-asset Instant Transaction (COMIT) is an open-source protocol for 
cross-blockchain applications, like peer-to-peer atomic swaps, and does not feature another blockchain nor token. It 
is powered by simple cryptographic principles such as HTLCs and enables users to trustlessly exchange one digital asset 
to another. Its specification, reference implementation in Rust, JavaScript Software Development Kit (SDK), CLI and web 
Graphical User Interface (GUI) are available on GitHub. At the time of writing, October&nbsp;2019, COMIT supports 
assets from the Bitcoin blockchain (e.g. BTC) and from the Ethereum blockchain (e.g. ETH and ERC20 tokens) 
([[11], [[12]], [[13]]). 
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 2]
```diff
@@ -82,6 +83,13 @@ as a third party. Although swaps have been done, it is stated as not yet being p
...
+Its reference implementation in rust as well as its specification are completly open source, free and publicly available on GitHub [[11],[14]]. Developers can take the provided JavaScript SDK [[15]] or get inspired by the provided CLI including examples [[16]] to build their own DEX.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 3]
```diff
@@ -82,6 +83,13 @@ as a third party. Although swaps have been done, it is stated as not yet being p
 (July&nbsp;2018), it is only possible to use Barterdex out of the Command Line Interface (CLI) [[9]]. Barterdex charges 
 a 0.1287% fee for a swap [[10]]. 
 
+### #4 COMIT
+COMIT is an open protocol for atomic swaps [[13]]. Compared to other solutions, COMIT does not feature another _blockchain_ nor token. Powered by simple cryptographic principles such as HTLCs [[12]] it enables users to trustlessly exchange one digital asset to another.
+Its reference implementation in rust as well as its specification are completly open source, free and publicly available on GitHub [[11],[14]]. Developers can take the provided JavaScript SDK [[15]] or get inspired by the provided CLI including examples [[16]] to build their own DEX. 
+
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 4]
```diff
@@ -82,6 +83,13 @@ as a third party. Although swaps have been done, it is stated as not yet being p
...
+Currently (October&nbsp;2019), COMIT supports assets from the Bitcoin blockchain (e.g. BTC) and assets from the Ethereum blockchain (e.g. ETH and ERC20 tokens).
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 5]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
 [10]: https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whitepaper-v2
 "barterDEX Whitepaper v2" 
 
+[[11]] COMIT (2018), "COMIT RFCs", GitHub [online].
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[[11]] GitHub: "comit-network, COMIT is an open protocol facilitating trustless cross-blockchain applications." 
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 6]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://github.com/comit-network/RFCs/>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[online]. Available: <https://github.com/comit-network>. Date accessed: 2019&#8209;10&#8209;16.
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 7]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[11]: https://github.com/comit-network/RFCs/
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[11]: https://github.com/comit-network
"GitHub: comit-network"
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 8]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[[12]] CoBloX (2018), "First atomic swap between Bitcoin and Ethereum" [online]
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[[12]] CoBloX (2018), "Connect all the Blockchains!!!" [online] Available: 
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 9]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://blog.coblox.tech/2018/06/23/connect-all-the-blockchains.html/>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
<https://blog.coblox.tech/2018/06/23/connect-all-the-blockchains.html/>. Date accessed: 2019&#8209;10&#8209;16.
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 10]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[12]: https://blog.coblox.tech/2018/06/23/connect-all-the-blockchains.html
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[12]: https://blog.coblox.tech/2018/06/23/connect-all-the-blockchains.html
"CoBloX Connect all the Blockchains!!!"
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 11]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[[13]] COMIT (2016), "COMIT is an open protocol facilitating trustless cross-blockchain applications." [online]
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[[13]] "C()MIT is an open protocol facilitating trustless cross-blockchain applications" [online]
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 12]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://comit.network/>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
Available: <https://comit.network/>. Date accessed: 2019&#8209;10&#8209;16.
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 13]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[13]: https://comit.network/
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
[13]: https://comit.network/
"C()MIT website"
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 14]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[[14]] COMIT (2018), "Reference implementation of COMIT." GitHub [online]
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 15]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://github.com/comit-network/comit-rs>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 16]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[14]: https://github.com/comit-network/comit-rs
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 17]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[[15]] COMIT (2019), "COMIT JavaScript SDK." GitHub [online]
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 18]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://github.com/comit-network/comit-js-sdk>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 19]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[15]: https://github.com/comit-network/comit-js-sdk
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 20]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[[16]] COMIT (2019), "CLI to setup a dev environment." GitHub [online]
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 21]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+Available: <https://github.com/comit-network/create-comit-app>. Date accessed: 2018&#8209;10&#8209;16.
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 22]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+
+[16]: https://github.com/comit-network/create-comit-app
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 23]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+[14]: https://github.com/comit-network/comit-rs
+
+[[15]] COMIT (2019), "COMIT JavaScript SDK." GitHub [online]
+Available: <https://github.com/comit-network/comit-js-sdk>. Date accessed: 2018&#8209;10&#8209;16.
+
+[15]: https://github.com/comit-network/comit-js-sdk
+
+[[16]] COMIT (2019), "CLI to setup a dev environment." GitHub [online]
+Available: <https://github.com/comit-network/create-comit-app>. Date accessed: 2018&#8209;10&#8209;16.
+
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 24]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+[[14]] COMIT (2018), "Reference implementation of COMIT." GitHub [online]
+Available: <https://github.com/comit-network/comit-rs>. Date accessed: 2018&#8209;10&#8209;16.
+
+[14]: https://github.com/comit-network/comit-rs
+
+[[15]] COMIT (2019), "COMIT JavaScript SDK." GitHub [online]
+Available: <https://github.com/comit-network/comit-js-sdk>. Date accessed: 2018&#8209;10&#8209;16.
+
+[15]: https://github.com/comit-network/comit-js-sdk
+
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 25]
```diff
@@ -145,7 +153,39 @@ Available: <https://github.com/KomodoPlatform/KomodoPlatform/wiki/barterDEX-Whit
...
+[12]: https://blog.coblox.tech/2018/06/23/connect-all-the-blockchains.html
+
+[[13]] COMIT (2016), "COMIT is an open protocol facilitating trustless cross-blockchain applications." [online]
+Available: <https://comit.network/>. Date accessed: 2018&#8209;10&#8209;16.
+
+[13]: https://comit.network/
+
+[[14]] COMIT (2018), "Reference implementation of COMIT." GitHub [online]
+Available: <https://github.com/comit-network/comit-rs>. Date accessed: 2018&#8209;10&#8209;16.
+
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 26]
```diff
@@ -7,6 +7,7 @@
   - [#1 Manual Method](#a1-manual-method)
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
  - [#1 Manual Method](#1-manual-method)
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 27]
```diff
@@ -7,6 +7,7 @@
...
   - [#2 Atomic Wallet](#a2-atomic-wallet)
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
  - [#2 Atomic Wallet](#2-atomic-wallet)
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 28]
```diff
@@ -7,6 +7,7 @@
...
   - [#3 BarterDEX](#a3-barterdex)
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
  - [#3 BarterDEX](#3-barterdex)
```

File: src/protocols/atomic-swaps/AtomicSwaps.md [hunk 29]
```diff
@@ -7,6 +7,7 @@
...
+  - [#4 COMIT](#a4-comit)
```
Comment (hansieodendaal) [inline src/protocols/atomic-swaps/AtomicSwaps.md]:
```suggestion
  - [#4 COMIT](#4-comit)
```

File: src/protocols/atomic-swaps/AtomicSwaps.md (hunk 13 above)
Comment (bonomat) [reply src/protocols/atomic-swaps/AtomicSwaps.md]: I've used `COMIT` instead of `C( )MIT` as it does not render nicely in text.

**Comment by bonomat** [comment]
thanks @hansieodendaal for the feedback. Please have another look.

**Comment by hansieodendaal** [comment]
Thanks @bonomat, all good. Merging into master.

**Pull request merged**